spec/unit/platform/query_helpers_spec.rb

```diff
@@ -114,6 +114,26 @@
   end
 end
 
+describe "Chef::Platform#supports_msi?" do
+  let(:key) { "System\\CurrentControlSet\\Services\\msiserver" }
+  let(:key_query_value) { 0x0001 }
+  let(:access) { key_query_value }
+  let(:error_code) { 0 }
+
+  it "returns false early when not on windows" do
+    allow(ChefConfig).to receive(:windows?).and_return(false)
+    expect(Chef::Platform.supports_msi?).to be false
+  end
+
+  it "returns true when the registry key exists" do
+    allow(ChefConfig).to receive(:windows?).and_return(true)
+    allow(Win32::Registry::HKEY_LOCAL_MACHINE).to receive(:open).
```

Comment:
Additionally, you'll either have to wait for [this PR](https://github.com/chef/chef/pull/3937) to be merged and do an `include_context "Win32"` [like this](https://github.com/chef/chef/blob/mcquin/remove-const/spec/unit/platform/query_helpers_spec.rb#L34-L35), or you need to include these [before](https://github.com/chef/chef/blob/master/spec/unit/platform/query_helpers_spec.rb#L41-L58) and [after](https://github.com/chef/chef/blob/master/spec/unit/platform/query_helpers_spec.rb#L60-L68) blocks.

It'd be easiest if you waited until the PR is merged. ;)